src/renderer/Switch.tsx

```diff
@@ -1,44 +1,56 @@
-import './Switch.css';
-import { useEffect, useContext, ChangeEventHandler } from 'react';
+import { KeyboardEvent, useEffect, useContext, useState } from 'react';
 import { PeaceFoundContext } from './PeaceFoundContext';
+import './styles/Switch.scss';
 
 interface ISwitchProps {
   isOn: boolean | undefined;
-  handleToggle: ChangeEventHandler<HTMLInputElement>;
+  handleToggle: () => void;
   id: string;
   onLoad: () => void;
 }
 
-// Taken from https://upmostly.com/tutorials/build-a-react-switch-toggle-component
+// Structure taken from https://upmostly.com/tutorials/build-a-react-switch-toggle-component
 export default function Switch({
   isOn,
   handleToggle,
   id,
   onLoad,
 }: ISwitchProps) {
   const { peaceError } = useContext(PeaceFoundContext);
+  const [isKeyDown, setIsKeyDown] = useState(false);
   useEffect(() => {
     if (!peaceError) {
       onLoad();
     }
   }, [onLoad, peaceError]);
 
+  const handleKeyDown = (e: KeyboardEvent) => {
+    if (e.code === 'Enter' && !isKeyDown) {
+      handleToggle();
```

Comment:
Shouldn't we only toggle on a keyUp event?

Reply:
Good call, will make the change